content/id/docs/concepts/storage/volume-snapshot-classes.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Volume Snapshot Class
+content_template: templates/concept
+weight: 30
+---
+
+{{% capture overview %}}
+
+Laman ini menjelaskan tentang konsep VolumeSnapshotClass pada Kubernetes. Sebelum melanjutkan,
+sangat disarankan untuk membaca [_snapshot_ volume](/docs/concepts/storage/volume-snapshots/)
```

Comment:
_snapshot_ volume -> volume _snapshot_

Reply:
I think for Indonesian language it's the opposite, it's like:
Language Class (en) -> Kelas Bahasa (id)

order becomes opposite